packages/backend/src/controllers/projectController.ts

```diff
@@ -505,4 +513,109 @@ export class ProjectController extends BaseController {
             results: undefined,
         };
     }
+
+    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @SuccessResponse('200', 'Success')
+    @Get('{projectUuid}/dashboards')
```

Comment:
Get methods should be safe to translate. 
Post methods, with Record<> can be dangerous and we need to manually test its backwards compatibility